eclair-core/src/test/scala/fr/acinq/eclair/db/AuditDbSpec.scala

```diff
@@ -566,9 +564,10 @@ class AuditDbSpec extends AnyFunSuite {
       val db = dbs.audit
       val sqlite = dbs.connection
       val isPg = dbs.isInstanceOf[TestPgDatabases]
+      val table = if (isPg) "audit.relayed" else "relayed"
 
-      using(sqlite.prepareStatement("INSERT INTO relayed (payment_hash, amount_msat, channel_id, direction, relay_type, timestamp) VALUES (?, ?, ?, ?, ?, ?)")) { statement =>
-        if (isPg) statement.setString(1, randomBytes32().toHex) else statement.setBytes(1, randomBytes32().toArray)
+      using(sqlite.prepareStatement(s"INSERT INTO $table (payment_hash, amount_msat, channel_id, direction, relay_type, timestamp) VALUES (?, ?, ?, ?, ?, ?)")) { statement =>
+        if (isPg) statement.setString(1, randomBytes32.toHex) else statement.setBytes(1, randomBytes32().toArray)
```

Comment:
nit: `randomBytes32()` (missing parenthesis)

Reply:
Done in 94db6f66e.